### DISCUSSION TITLE
Where can I find console.keys?

### DISCUSSION THREAD

**Discussion by 1DontEx1st** [a1ex4/ownfoil#127] [General]
Are "console keys" different from prod.keys / title.keys?

**Comment by Masamune3210** [comment]
console keys = prod keys. Prod is just short for production, i.e., the normal model sold to users. its similar twin, dev.keys, is for development units, the ones sold to devs and publishers for use with making games.

**Reply by televisi** [reply]
Thanks, I was wondering what to put into the UI.
I selected "prod.keys" file and it works now
